src/app/containers/RelatedTopics/index.stories.jsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import { withKnobs } from '@storybook/addon-knobs';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { ToggleContextProvider } from '#contexts/ToggleContext';
+import { EventTrackingContextProvider } from '#contexts/EventTrackingContext';
+import { STORY_PAGE } from '#app/routes/utils/pageTypes';
+import RelatedTopics from '.';
+
+const WithContexts = ({
+  children,
+  variant,
+  enabled = true,
+  service = 'mundo',
+}) => {
+  return (
+    <RequestContextProvider
+      service={service}
+      variant={variant}
+      pageType={STORY_PAGE}
+      isAmp={false}
+      pathname="/"
+    >
+      <ServiceContextProvider service={service}>
+        <ToggleContextProvider
+          toggles={{
+            eventTracking: {
+              enabled: true,
+            },
+            topicsTags: {
+              enabled,
+            },
+          }}
+        >
+          <EventTrackingContextProvider>
+            {children}
+          </EventTrackingContextProvider>
+        </ToggleContextProvider>
+      </ServiceContextProvider>
+    </RequestContextProvider>
+  );
+};
+
+storiesOf('Containers/RelatedTopics', module)
```

Comment:
It might be worth working @ryanmccombe to write these using the 'modern approach' to writing stories and he is in process of updating them across the board for the storybook upgrade?